tasks/windows.task/second-stage.ps1.erb

```diff
@@ -40,5 +41,5 @@ New-Item -ItemType Directory -Force -Path "$($osDrive)\Windows\Setup\Scripts"
 (New-Object System.Net.WebClient).DownloadFile('<%= file_url('SetupComplete.cmd') %>',"$($osDrive)\Windows\Setup\Scripts\SetupComplete.cmd")
 
 write-host "restarting the system to complete installation"
-restart-computer
-start-sleep 30                  # wait for the reboot to complete before exiting
+# wait for the reboot to complete before exiting
+restart-computer -Wait -For PowerShell -Timeout 30
```

Comment:
I see no harm in bumping up the amount of time that this waits. I'll make it a minute. Any more than that, the user can always clone and modify this file as a custom task (which most places will likely want to do anyway to customize their unattended.xml).